compute/src/test/java/org/jclouds/compute/ComputeTestUtils.java

```diff
@@ -62,7 +64,18 @@
    public static void checkSecretKeyFile(String secretKeyFile) {
       checkNotNull(emptyToNull(secretKeyFile), "System property: [test.ssh.keyfile] set to an empty string");
       if (!new File(secretKeyFile).exists()) {
-         throw new IllegalStateException("secretKeyFile not found at: " + secretKeyFile);
+         // Create file if missing
+         try {
+            JSch jsch = new JSch();
+            KeyPair kPair = KeyPair.genKeyPair(jsch, KeyPair.RSA, 4096);
+            kPair.writePrivateKey(secretKeyFile);
+            kPair.writePublicKey(secretKeyFile + ".pub", "jclouds generated key pair");
+            kPair.dispose();
```

Comment:
I notice we generate a 2048 key... Any issues upping that to 4k?
Also, yes, great point - it's possible to pass the keypair directly, but I am not sure the files are used after during testing or not. Still checking.